app/post.go

```diff
@@ -396,6 +397,11 @@ func (a *App) FillInPostProps(post *model.Post, channel *model.Channel) *model.A
 		post.DelProp("channel_mentions")
 	}
 
+	matched, err := regexp.MatchString(`\B@`, post.Message)
+	if err == nil && matched && !a.HasPermissionToChannel(post.UserId, post.ChannelId, model.PERMISSION_USE_GROUP_MENTIONS) {
```

Comment:
A license check here makes sense i think so that this group highlight disabled prop is only added when the server has ldap groups

Reply:
Done! Thanks